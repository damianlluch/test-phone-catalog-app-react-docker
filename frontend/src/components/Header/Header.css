.Header {
  display: flex;
  width: 100%;
  justify-content: center;
  padding: 20px;
  background-color: #3C4043;
  margin-bottom: 20px;
}

.Header a {
  color: #F1F3F4;
  text-decoration: none;
}

.Header h1 {
  font-size: 1.7em;
}

@media (max-width: 1000px){
	.Header{
    padding: 18px;
    margin-bottom: 18px;
  }
}

@media (max-width: 600px){
	.Header{
    padding: 16px;
    margin-bottom: 16px;    
  }
}
