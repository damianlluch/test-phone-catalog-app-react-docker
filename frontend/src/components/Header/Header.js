import React from 'react';
import { Link } from 'react-router-dom';
import './Header.css';

const Header = () => (
  <header className="Header">
    <Link to="/">
      <h1>Phone Catalog App</h1>
    </Link>
  </header>
);

export default Header;
