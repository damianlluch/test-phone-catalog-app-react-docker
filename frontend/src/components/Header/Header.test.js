import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import Header from './Header';

describe('<Header/>', () => {
  it('renders by default', () => {
    const tree = renderer.create(<MemoryRouter><Header /></MemoryRouter>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
