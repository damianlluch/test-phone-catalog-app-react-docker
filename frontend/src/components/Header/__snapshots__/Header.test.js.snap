// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header/> renders by default 1`] = `
<header
  className="Header"
>
  <a
    href="/"
    onClick={[Function]}
  >
    <h1>
      Phone Catalog App
    </h1>
  </a>
</header>
`;
